esrally/mechanic/telemetry.py

```diff
@@ -47,16 +47,12 @@ def __init__(self, enabled_devices=None, devices=None):
         self.devices = devices
 
     def instrument_candidate_env(self, car, candidate_id):
-        opts = {}
+        opts = []
         for device in self.devices:
             if self._enabled(device):
                 additional_opts = device.instrument_env(car, candidate_id)
```

Comment:
Similarly we should rename `instrument_env` of the individual telemetry devices.